velox/exec/tests/SparkAggregationFuzzerTest.cpp

```diff
@@ -71,6 +71,10 @@ int main(int argc, char** argv) {
   size_t initialSeed = FLAGS_seed == 0 ? std::time(nullptr) : FLAGS_seed;
   auto duckQueryRunner =
       std::make_unique<facebook::velox::exec::test::DuckQueryRunner>();
+  duckQueryRunner->disableAggregateFunctions({
+      "max_by", // DuckDB results on equality handling is different with Spark.
```

Comment:
> // DuckDB results on equality handling is different with Spark.

I don't understand this comment and wonder if other readers will be able to understand it. Perhaps, replace it with a link to GitHub issue that provides more context.

```
  // https://github.com/facebookincubator/velox/issues/7677
  "max_by",
  "min_by",
```

Reply:
Fixed, thanks.